src/guardian/staking-module-guard/staking-module-guard.service.ts

```diff
@@ -156,6 +227,43 @@ export class StakingModuleGuardService {
     return attackIntersections;
   }
 
+  public async getIntersectionBetweenUsedAndUnusedKeys(
+    intersectionsWithLidoWC: VerifiedDepositEvent[],
+    blockData: BlockData,
+  ) {
+    const depositedPubkeys = intersectionsWithLidoWC.map(
+      (deposit) => deposit.pubkey,
+    );
+
+    if (depositedPubkeys.length) {
```

Comment:
I would suggest inverting the condition and making an early exit

Reply:
okay